### DISCUSSION TITLE
Module 6-1 Cannot create a new workspace

### DISCUSSION THREAD

**Discussion by wtb4rtr** [ua-gist604b-f24/syllabus#28] [General]
Hi I have had problems in Module 6 being denied access to geoserver when I tried to 1) create the custom symbology for the archaeological sites (Module 6.0) and creating a new workspace for Module 6.1.  I really hope I can get my workspace created because I cannot do any of the module without it.  Below is a screencap of the error message I keep getting.  Thanks in advance for any help
Bill Brown
![screencap-geoserver-cant-access-publishing](https://github.com/user-attachments/assets/40f42da8-9a88-42a1-9c95-6c8a0d2860c3)

**Comment by andreasrf** [comment]
It seems like a bunch of folks are having issues with this module. Hang in there, Bill! This might be a long shot, but maybe try setting the visibility of your port to Public if it isn't already? Also make sure your Codespace is still running, I've made that mistake before. If you have VPNs on you also might want to turn those off. That's about all I can think of to help you.

**Reply by wtb4rtr** [reply]
Thank you Andrea for responding; my port 8080 is set to public but I don't think that is the problem as I have tried just going to Geoserver and doing it and I get the same error message.  But, I really appreciate your help, good luck with the rest of the quarter
Bill Brown

**Reply by jimmy-nguyen-data-science** [reply]
Hi, I am having the same problem and cannot figure out how to create my workspace, please help!
